entrypoint.d/45-geoip

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
+    set -e
+    # see https://dev.maxmind.com/geoip/geoipupdate/
+    log INFO "Activating GeoIP/GeoLite2 updates"
+    log INFO "This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com"
+    sed 's/^AccountID\s.*$/AccountID '"$GEOIP_ACCOUNT_ID"'/g;s/^LicenseKey\s.*$/LicenseKey '"$GEOIP_LICENSE_KEY"'/g' /etc/GeoIP.conf > /tmp/GeoIP.conf
+    cat /tmp/GeoIP.conf > /etc/GeoIP.conf
+    rm /tmp/GeoIP.conf
+fi
+
+if [[ "$(grep 'AccountID' /etc/GeoIP.conf | sed 's/AccountID //')" != "0" ]] ; then
```

Comment:
Sorry about this, I'm just not familiar with the other syntax and I prefer to keep it consistent with the rest of the codebase.

```suggestion
if [ "$(grep 'AccountID' /etc/GeoIP.conf | sed 's/AccountID //')" != "0" ] ; then
```

Yikes, I should have moved to plumbum long ago here...

Reply:
I'm having a hard time understanding this `if`. Could you explain what you are trying to do please?

I'm tired of bash :man_facepalming: , when you explain it to me I'm going to upload a commit translating this into plumbum and you review it please. Thanks!

Reply:
Searching the line in /etc/GeoIP.conf (to be moved) that contains "AcountID " getting rid of the "AccountID " part, and checking if the set AccountID is not 0 (the default value)
`AccountID 1234` is the line when it is set, `AccountID 0` is the line when it is not set